### PR TITLE
Auto Placement LEDs

### LINKED ISSUE #141: LEDs when disabled to communicate placement for auto prior to a match (Alice's idea, https://youtu.be/aRY7PFJ0PF0?si=lLZTNuQqeonhHped)

### PR DESCRIPTION
**Pull request by AlenaNguyen**

### CODE REVIEW

File: src/main/java/frc/robot/RobotPreferences.java [hunk 1]
```diff
@@ -373,6 +373,22 @@ public static final class prefVision {
         "visionStdDevsHeading", Units.metersToFeet(0.9));
 
     public static final SN_DoublePreference maxAmbiguity = new SN_DoublePreference("maxAmbiguity", 0.2);
+
+    /**
+     * The translational tolerance of how off we want to be to count as correct
+     * (when placing the robot on the starting position in auto)
+     * <b>Units:</b> TODO: idk i need to check
```
Comment (alicekuznetsov) [inline src/main/java/frc/robot/RobotPreferences.java]: Probably meters

**Comment by alicekuznetsov** [comment]
I think we're good to merge this in
Getting the pose from the auto will be done when autos are added - we want this in main
@TaylerUva  thoughts?

**Comment by TaylerUva** [comment]
> I think we're good to merge this in
> 
> Getting the pose from the auto will be done when autos are added - we want this in main
> 
> @TaylerUva  thoughts?

Sounds good. However,
There are merge conflicts @ACat701

**Comment by alicekuznetsov** [comment]
> > I think we're good to merge this in
> > Getting the pose from the auto will be done when autos are added - we want this in main
> > @TaylerUva  thoughts?
> 
> Sounds good. However, There are merge conflicts @ACat701

github when imports 😨

**Review by TaylerUva — APPROVED** [review]

**Pull request merged**